src/hearings/resolvers/priority/priority.resolver.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { RouterStateSnapshot } from '@angular/router';
+import { of } from 'rxjs';
+import { RefDataModel } from './../../../../api/hearings/models/refData.model';
```

Comment:
copy api/hearings/models/refData.model into src/hearings/models/refData.model, node models and ng models need to be separated.